xwiki-commons-core/xwiki-commons-environment/xwiki-commons-environment-standard/src/main/java/org/xwiki/environment/internal/StandardEnvironment.java

```diff
@@ -47,7 +47,7 @@
     /**
      * @see #setResourceClassLoader(ClassLoader)
      */
-    private ClassLoader resourceClassLoader = StandardEnvironment.class.getClassLoader();
+    private ClassLoader resourceClassLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
As I told you in the chat this is not the right way. You should use Thread.currentThread().getContextClassLoader() where is needed and not use it to initialize a field.